modules/openapi-generator/src/main/resources/typescript-fetch/runtime.mustache

```diff
@@ -137,7 +137,7 @@ export class Configuration {
         return this.configuration.basePath != null ? this.configuration.basePath : BASE_PATH;
     }
 
-    get fetchApi(): FetchAPI {
+    get fetchApi(): FetchAPI | undefined {
         return this.configuration.fetchApi;
```

Comment:
this should be changed to `return this.configuration.fetchApi || window.fetch.bind(window);` and not typed as if it could be undefined

Reply:
But BaseApi [already provides a fallback here](https://github.com/javier-garcia-meteologica/openapi-generator/blob/fix_typescript_fetch_fetchApi/modules/openapi-generator/src/main/resources/typescript-fetch/runtime.mustache#L78), so that every call to `this.fetchApi` has the global `fetch` as a fallback.

If a default value is implemented in the configuration, that other fallback should be removed?

Reply:
The previous PR (https://github.com/OpenAPITools/openapi-generator/pull/10072) had good reasons to avoid `window` , in order to support Node environment (as in universal rendering)

Reply:
I've changed the code to address both issues. `fetchApi` returns always a defined `FetchAPI` and uses global `fetch` as fallback

Reply:
Oh, I see  this has already been fixed in  #10216, I will close this PR